spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/mkldnn/Fusion.scala

```diff
@@ -122,14 +134,30 @@ private[mkldnn] object Fusion {
       if (conv != null) {
         node.element = conv.element
         val element = node.element.asInstanceOf[SpatialConvolution]
-        element.setSumOp(previousNodes(otherNumber).element, otherNumber + 1)
+        element.setSumOp(previousNodes(otherNumber).element, otherNumber)
```

Comment:
why delete "+1" operation?

Reply:
OK. I have changed back. For original version there's a bug. And currently, there's no.